### PR TITLE
magento/magento2#23063: Translate product email alert

### PR DESCRIPTION
**Pull request by lavanyadckap**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/ProductAlert/Model/Observer.php [hunk 1]
```diff
@@ -332,7 +332,7 @@ protected function _processStock(\Magento\ProductAlert\Model\Email $email)
                     $product = $this->productRepository->getById(
                         $alert->getProductId(),
                         false,
-                        $website->getDefaultStore()->getId()
+                        ($alert->getStoreId())?$alert->getStoreId():$website->getDefaultStore()->getId()
```
Comment (lbajsarowicz) [inline app/code/Magento/ProductAlert/Model/Observer.php:335]:
```suggestion
                        $alert->getStoreId() ?: $website->getDefaultStore()->getId()
```

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Please find the comment attached and cover change with Unit / Integration tests.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Could you apply the suggestion?

**Comment by engcom-Charlie** [comment]
Hi @lavanyadckap, I'm closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for your collaboration.

**Pull request closed without merging**